frontend/src/model/beta/reviews/ReviewDisplay.tsx

```diff
@@ -36,7 +36,7 @@ export default function ReviewDisplay({ review }: ReviewDisplayProps) {
             )}
             <Typography variant='caption'>
               <Box component='span' fontWeight='bold'>
-                {reviewResponse.user.split(':')[isAccepted ? 1 : 0]}
+                {reviewResponse.user.split(':')[1]}
```

Comment:
Why has this logic changed?

Reply:
This will always be the second part of the entity string, this was fixing a bug rather than changing logic